src/web/nextui/src/app/eval/store.ts

```diff
@@ -79,10 +88,22 @@ export const useStore = create<TableState>()(
 
       inComparisonMode: false,
       setInComparisonMode: (inComparisonMode: boolean) => set(() => ({ inComparisonMode })),
+
+      columnStates: {},
+      setColumnState: (evalId: string, state: ColumnState) =>
+        set((prevState) => ({
+          columnStates: {
+            ...prevState.columnStates,
+            [evalId]: state,
+          },
+        })),
     }),
     {
       name: 'ResultsViewStorage',
       storage: createJSONStorage(() => storage),
+      partialize: (state) => ({
```

Comment:
Pretty sure this omits all the other properties from storage, which is not what we want.  The entire state object is stored by default.